plugins/modules/remote_management/manageiq/manageiq_policies_info.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022, Alexei Znamensky <russoz@gmail.com>
+# Copyright (c) 2017, Daniel Korn <korndaniel1@gmail.com>
+# Copyright (c) 2017, Yaacov Zamir <yzamir@redhat.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+
+module: manageiq_policies_info
+version_added: 5.7.0
+
+short_description: Listing of resource policy_profiles in ManageIQ
+extends_documentation_fragment:
+  - community.general.manageiq
+
+author: Alexei Znamensky (@russoz)
+description:
+  - The manageiq_policies module supports listing policy_profiles in ManageIQ.
+
+options:
+  resource_type:
+    type: str
+    description:
+      - The type of the resource to obtain the profile for.
+    required: true
+    choices: ['provider', 'host', 'vm', 'blueprint', 'category', 'cluster',
+        'data store', 'group', 'resource pool', 'service', 'service template',
+        'template', 'tenant', 'user']
+  resource_name:
+    type: str
+    description:
+      - The name of the resource to obtain the profile for.
+      - Must be specified if I(resource_id) is not set. Both options are mutually exclusive.
+  resource_id:
+    type: int
+    description:
+      - The ID of the resource to obtain the profile for.
+      - Must be specified if I(resource_name) is not set. Both options are mutually exclusive.
+'''
+
+EXAMPLES = '''
+- name: List current policy_profile and policies for a provider in ManageIQ
+  community.general.manageiq_policies_info:
+    resource_name: 'EngLab'
+    resource_type: 'provider'
+    manageiq_connection:
+      url: 'http://127.0.0.1:3000'
+      username: 'admin'
+      password: 'smartvm'
+      validate_certs: false
+  register: result
+'''
+
+RETURN = '''
+manageiq_policies:
+    description:
+      - List current policy_profile and policies for a provider in ManageIQ
+    returned: always
+    type: dict
```

Comment:
From the example, I think this would be more correct:
```suggestion
    type: list
    elements: dict
```